src/sensitivities/linalg/factorization/svd.jl

```diff
@@ -28,6 +28,13 @@ function ∇(::typeof(getproperty), ::Type{Arg{1}}, p, y, ȳ, USV::SVD, x::Symb
     end
 end
 
+# Iteration allows destructuring, e.g. U, S, V = svd(A)
+# These definitions mirror those defined in the LinearAlgebra module
```

Comment:
Link / permalink?

Reply:
I guess, doesn't seem that useful to me, but if you think it is then I can add it. It's just the same code but with `SVD` instead of `Branch{<:SVD}`.

Reply:
Definitely optional. My thoughts were if we're mirroring code elsewhere having a permalink can help track future devs determine what you were copying and apply future changes to LinearAlgebra

Reply:
Okay, that makes sense.